src/pages/Form/AllForms.tsx

```diff
@@ -17,8 +18,22 @@ const AllForms = () => {
     jwt,
     searchString: searchQuery,
   });
+  const {data: applicationForm} = useGetApplicationFormQuery(jwt);
 
-  const forms=data?.forms;
+  console.log("application form ----", applicationForm)
+  
+  const parsedApplicationForm = {
+    _id: "6707a1241caed33cd7c941dc",
+    name: applicationForm?.name,
+    description: applicationForm?.description,
+    type: applicationForm?.type,
+    questions: applicationForm?.questions.length,
+    startDate: applicationForm?.startDate
+  }
+  const forms = data?.forms;
```

Comment:
I remember this is because the backend does not return the Id of the form.
But Let's fix that so that we don't merge this...

Reply:
True true